docs/manuals/core/entities/scenario-cycle-mgt.md

```diff
@@ -89,6 +89,11 @@ The `taipy.get_primary()^` method returns the primary scenario of the cycle give
 
 `taipy.get_primary_scenarios()^` returns the primary scenarios for all the existing cycles.
 
+# Get all scenarios grouped by cycles
```

Comment:
Should this be moved right after `# Get all scenarios`?
Because both is to retrieved all scenarios.